### DISCUSSION TITLE
BTT TFT35 - Double extruder - Ok in marlin mode and not ok in touch mode

### DISCUSSION THREAD

**Discussion by Jm5964** [bigtreetech/BIGTREETECH-TouchScreenFirmware#2353] [Q&A] [answered]
Hello, 
I'm having a problem with BBT's TFT35 screen. 
In Marlin mode, I have the possibility to heat my 2 nozzles
In Touch mode, I only see one nozzle T0.  

In configuration.h : 
#define HOTEND_COUNT 2
#define EXTRUDER_COUNT 2

in config.ini : 
ext_count:2

thanks a lot for your help and sorry for the bad translate

Version firmware BIGTREE_TFT35_V3.0.27.x.bin

**Comment by radek8** [comment]
Can you enter the answer on M115 and M503?

**Comment by Jm5964** [comment, marked as answer]
Hello Radek8,

The problem is solved.
I put the firmware and the config.ini again and now it's ok.  Thank you for taking an interest in my problem Good evening.
